L1Trigger/L1THGCal/python/hgcalConcentratorProducer_cfi.py

```diff
@@ -5,14 +5,34 @@
 adcSaturationBH_MIP = digiparam.hgchebackDigitizer.digiCfg.feCfg.adcSaturation_fC
 adcNbitsBH = digiparam.hgchebackDigitizer.digiCfg.feCfg.adcNbits
 
+EE_DR_GROUP = 7
+FH_DR_GROUP = 6
+BH_DR_GROUP = 12
+MAX_LAYERS = 52
+
+STC_SIZE_CUSTOM_PERLAYER_100 = ([4] + # no layer 0
+                     [4]*EE_DR_GROUP + [4]*EE_DR_GROUP + [4]*EE_DR_GROUP + [4]*EE_DR_GROUP + # EM
+                     [4]*FH_DR_GROUP + [4]*FH_DR_GROUP + # FH
+                     [4]*BH_DR_GROUP) # BH
+STC_SIZE_CUSTOM_PERLAYER_200 = ([16] + # no layer 0
+                     [16]*EE_DR_GROUP + [16]*EE_DR_GROUP + [16]*EE_DR_GROUP + [16]*EE_DR_GROUP + # EM
+                     [16]*FH_DR_GROUP + [16]*FH_DR_GROUP + # FH
+                     [16]*BH_DR_GROUP) # BH
+STC_SIZE_CUSTOM_PERLAYER_300 = STC_SIZE_CUSTOM_PERLAYER_200
+STC_SIZE_CUSTOM_PERLAYER_SCIN = STC_SIZE_CUSTOM_PERLAYER_200
+
+
+CTC_SIZE =  ( [2]*(MAX_LAYERS+1)*4)
+STC_SIZE =  ( [4]*(MAX_LAYERS+1)+ [16]*(MAX_LAYERS+1)*3 )
+
 
 threshold_conc_proc = cms.PSet(ProcessorName  = cms.string('HGCalConcentratorProcessorSelection'),
                                Method = cms.string('thresholdSelect'),
                                threshold_silicon = cms.double(2.), # MipT
                                threshold_scintillator = cms.double(2.), # MipT
                                coarsenTriggerCells = cms.bool(False),
                                fixedDataSizePerHGCROC = cms.bool(False),
-                               ctcSize = cms.vuint32(2,2,2,2),
+                               ctcSize = cms.vdouble(CTC_SIZE),
```

Comment:
Is there a reason to have now a vector of double instead of int?

Reply:
no, fixed